staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -506,6 +506,9 @@ func RunTestList(ctx context.Context, t *testing.T, store storage.Interface, ign
 		pod := obj.(*example.Pod)
 		return nil, fields.Set{"metadata.name": pod.Name, "spec.nodeName": pod.Spec.NodeName}, nil
 	}
+	// Bump resource version in etcd in a way not observed by watch cache. Used for consistent List test.
```

Comment:
please explain in more details (i.e. that compaction underneath writes something to a "hidden key").

Reply:
Done